test/e2e-cypress/e2e/features/response-empty-examples-object.cy.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @prettier
+ */
+
+describe("Response examples", () => {
+  it("should render a generated example when an empty examples object is provided", () => {
+    cy.visit("/?url=/documents/features/response-empty-examples-object.yaml")
+      .get("#operations-TEST-test")
+      .click()
+      .get(".example.microlight")
+      .contains("{}")
+      .should("exist")
```

Comment:
Just a little bit more logical order
```suggestion
      .should("exist")
      .contains("{}")
```